### PR TITLE
Bug 1776141, NSX-T VXLAN port conflict

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1776141

### CODE REVIEW

**Comment by codyhoag** [comment]
@lihongan can you confirm? Thanks!

**Comment by lihongan** [comment]
Looks the QE owner of this issue should be SDN team.
@zhaozhanqi @huiran0826 @anuragthehatter can anyone help review?

**Comment by huiran0826** [comment]
LGTM

File: modules/nw-operator-cr.adoc [hunk 1]
```diff
@@ -152,7 +152,10 @@ then you must set this explicitly to 50 less than the smallest node MTU value.
 
 <5> The port to use for all VXLAN packets. The default value is `4789`. If you
 are running in a virtualized environment with existing nodes that are part of
-another VXLAN network, then you might be required to change this.
+another VXLAN network, then you might be required to change this. For example,
+when running an {product-title} SDN overlay on top of VMware NSX-T, you must
```
Comment (kalexand-rh) [inline modules/nw-operator-cr.adoc]: s/{product-title}/OpenShift

**Pull request merged**